packages/stack/contracts-manager/src/index.js

```diff
@@ -325,7 +325,7 @@ export default class ContractsManager {
           contract.swarmHash = compiledContract.swarmHash;
           contract.gasEstimates = compiledContract.gasEstimates;
           contract.functionHashes = compiledContract.functionHashes;
-          contract.abiDefinition = compiledContract.abiDefinition;
+          contract.abiDefinition = contractConfig && contractConfig.abiDefinition ? contractConfig.abiDefinition : compiledContract.abiDefinition;
```

Comment:
Prettier with optional chaining
```suggestion
          contract.abiDefinition = contractConfig?.abiDefinition ? contractConfig.abiDefinition : compiledContract.abiDefinition;
```